.gitignore

```diff
@@ -83,6 +83,10 @@ celerybeat-schedule
 # dotenv
 .env
 
+# direnv
```

Comment:
I think this was added accidentally, since it is not explicitly about upgrading to version 5.

Reply:
I'll separate it.